src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/codeAction/proposal/quickfix/RemoveAnnotationConflictQuickFix.java

```diff
@@ -41,11 +50,15 @@
  * @author Angelo ZERR
  *
  */
-public class RemoveAnnotationConflictQuickFix {
-    private final String[] annotations;
+public abstract class RemoveAnnotationConflictQuickFix implements IJavaCodeActionParticipant {
 
+    protected static final String ANNOTATION_LIST = "annotationList";
+    private static final String ANNOTATION_TO_REMOVE = "annotationsToRemove";
+    private String[] annotations;
```

Comment:
I'm curious why this field is no longer final. We should only be initializing this in the constructor and it should be read-only from that point.